src/components/TodoList/TodoList.tsx

```diff
@@ -0,0 +1,21 @@
+import { Todo } from '../../types/Todo';
+import { TodoItem } from '../TodoItem/TodoItem';
+
+type Props = {
+  todos: Todo[] | null,
+  removeTodo: (vatodoIdlue: number) => void,
+};
+
+export const TodoList:React.FC<Props> = ({ todos, removeTodo }) => {
+  return (
+    <>
+      {todos?.map(todo => (
+        <TodoItem
+          todo={todo}
+          key={todo.id}
+          removeTodo={removeTodo}
+        />
```

Comment:
There should be conditional rendering for `TodoItem `for `tempTodo`, while you're waiting for response when new todo is creating.